src/hooks/comments/service.ts

```diff
@@ -0,0 +1,13 @@
+import { axiosClient } from '@/apis';
+import { formatOptionQuery, IFormatOptionQuery } from '@/utils/getPagiantion';
+import { format } from 'path';
+import { PaginationServicer } from '../interfaces';
+import { ResponseGetComments } from './interface';
+
+class CommentService implements PaginationServicer<ResponseGetComments> {
+  basePublicURL: string = '/public/comments';
+  getPagination(query: IFormatOptionQuery): Promise<ResponseGetComments> {
```

Comment:
I think it should be name like `getComments` or `fetchComments` instead of `getPagination`, it makes me confused